commit f825bd28e15f4a4e653f8a81e2a02ac9da1068ca
Author: janluca <janluca@zedat.fu-berlin.de>
Date:   Sun Dec 27 14:34:04 2015 +0100

    MMC: Do not use mmc_debug if CONFIG_MMC_BCM2835 is not set
    
    If CONFIG_MMC_BCM2835 was not set the compiling of the kernel failed
    since mmc_debug was not defined but used in drivers/mmc/core/quirks.c.
    
    This patch add a ifdef-check for CONFIG_MMC_BCM2835 to the change of
    commit 64d395457f793250d2e582eeb38cc3403b1db98c

diff --git a/drivers/mmc/core/quirks.c b/drivers/mmc/core/quirks.c
index 87ae2e9..b79fe14 100644
--- a/drivers/mmc/core/quirks.c
+++ b/drivers/mmc/core/quirks.c
@@ -53,7 +53,9 @@ static const struct mmc_fixup mmc_fixup_methods[] = {
 
 void mmc_fixup_device(struct mmc_card *card, const struct mmc_fixup *table)
 {
+#ifdef CONFIG_MMC_BCM2835
 	extern unsigned mmc_debug;
+#endif
 	const struct mmc_fixup *f;
 	u64 rev = cid_rev_card(card);
 
@@ -81,7 +83,9 @@ void mmc_fixup_device(struct mmc_card *card, const struct mmc_fixup *table)
 	/* SDHCI on BCM2708 - bug causes a certain sequence of CMD23 operations to fail.
 	 * Disable this flag for all cards (fall-back to CMD25/CMD18 multi-block transfers).
 	 */
+#ifdef CONFIG_MMC_BCM2835
 	if (mmc_debug & (1<<13))
 	card->quirks |= MMC_QUIRK_BLK_NO_CMD23;
+#endif
 }
 EXPORT_SYMBOL(mmc_fixup_device);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1119_f825bd28e15f4a4e653f8a81e2a02ac9da1068ca.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1119_f825bd28e15f4a4e653f8a81e2a02ac9da1068ca.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
